src/RelayFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace Relay;
+
+class RelayFactory
+{
```

Comment:
I think it would be good to have `@var` declarations for the `$queue` and `$resolver`.